docs/features/azure-iam/tutorials/azure-iam-aks.mdx

```diff
@@ -310,7 +310,7 @@ By clicking on the `client` node, you can view and download the full set of Clie
 Use the otterize CLI to download and apply the intents to the cluster:
 
 ```bash
-otterize clientintents export -n otterize-tutorial-azure-iam.<CLUSTER NAME> -v v2 | kubectl apply -f-
```

Comment:
This is typically not necessary when users try this out on a fresh env (it'll only be required when multiple namespaces named `otterize-tutorial-azure-iam`). The cluster name here is the one used in otterize-cloud, so we can't guess it / use env vars for it. So better if we just omit it to allow easy copy-paste. The CLI will alert if there's some name conflict.